lib/logflare/alerting.ex

```diff
@@ -192,6 +191,33 @@ defmodule Logflare.Alerting do
     end)
   end
 
+  def sync_alert_jobs do
+    init_alert_jobs()
+    |> Enum.each(fn job ->
+      remote_call_scheduler(:add_job, [job])
+    end)
+  end
```

Comment:
Might be worth noting that this is used for testing specifically.

Btw. should this call `AlertScheduler.delete_all_jobs` beforehand, otherwise removed jobs are kept around?